website/client/src/components/groups/groupFormModal.vue

```diff
@@ -94,6 +94,46 @@ label.custom-control-label(v-once) {{ $t('allowGuildInvitationsFromNonMembers')
 // "allowGuildInvitationsFromNonMembers": "Allow Guild invitations from non-members",
         -->
       </div>
+      <div
+        v-if="isAdmin && workingGroup.id"
```

Comment:
In relation to the request to only allow the ability to change bannedWordsAllowed within updateGroup. I've included this conditional to ensure the banned words section/checkbox is only displayed on the update guild form modal